src/openzaak/components/zaken/admin/zaken.py

```diff
@@ -806,3 +828,31 @@ def get_queryset(self, request):
                 )
             )
         )
+
+    def save_model(self, request, obj, form, change):
+        super().save_model(request, obj, form, change)
+
+        if (
+            change
+            and form.changed_data == ["laatst_geopend"]
+            and settings.ENABLE_CLOUD_EVENTS
+        ):
+            process_cloudevent(
+                type=ZAAK_GEOPEND,
+                subject=str(obj.uuid),
+                data={
+                    "identificatie": obj.identificatie,
+                    "laatst_geopend": obj.laatst_geopend.isoformat(),
+                },
+            )
```

Comment:
I just now realized this field is read-only via the admin :facepalm:, so let's exclude this
```suggestion
```